web/test/spec/RaportitSpec.js

```diff
@@ -0,0 +1,26 @@
+describe('Raporttien luominen', function() {
+  var page = RaportitPage()
+  var login = LoginPage()
+
+  describe('Tietoturva', function() {
+    before(Authentication().logout, page.openPage())
+
+    it('näytetään login, kun käyttäjä ei ole kirjautunut sisään', function() {
+      expect(login.isVisible()).to.equal(true)
+    })
+  })
+
+  describe('Avataan organisaatiovalitsin', function() {
+    before(Authentication().login('kalle'), page.openPage(), page.avaaOrganisaatioValitsin())
+    it('Organisaatiohaku avautuu', function () {
+      expect(page.organisaatioHakuNäkyvissä()).to.equal(true)
+    })
+    // Poista kommentit, tää testi takas
+    /*describe('Valitaan organisaatio ja luodaan "Esiopetuksen opiskeluoikeus- ja suoritustietojen tarkistusraportti" raportti', function() {
+      before(page.valitseOrganisaatio('Jyväskylän normaalikoulu'), page.syötäPäivämäärä('1.1.2007'), page.luoRaportti())
```

Comment:
`page.organisaatioHakuNäkyvissä()).to.equal(true)` pitäisi mielestäni tehdä (myös) sisemmän describe:n before:ssa: muuten sisemmän describe:n it-testiä ei voi ajaa luotettavasti yksinään, kun hakukenttä ei välttämättä ole vielä auki.